packages/fluentui/projects-test/src/createReactApp.ts

```diff
@@ -34,7 +34,7 @@ export async function createReactApp() {
 
   logger('STEP 2. Add Fluent UI dependency to test project..');
 
-  const packedPackages = await packProjectPackages(logger, config.paths.packages(), ['@fluentui/react-northstar']);
+  const packedPackages = await packProjectPackages(logger, config.paths.base(), ['@fluentui/react-northstar']);
```

Comment:
As we have a package outside `packages/fluentui` I had to change the root to include `@fluentui/react-portal-compat-context` into the list of packages to build.